src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/MandateForChangeTest.kt

```diff
@@ -20,11 +27,11 @@ class MandateForChangeTest : MockedEndpointsTestBase() {
     setupRestrictiveRequirements(crn)
     setupMaleOffenderWithRegistrations(crn)
 
-    val expectedTierUpdate = tierUpdateWillSucceed(crn, "A0")
-
     listener.listen(calculationMessage(crn))
+    val tier = repo.findFirstByCrnOrderByCreatedDesc(crn)
 
-    mockCommunityApiServer.verify(expectedTierUpdate)
+    Assertions.assertThat(tier?.data?.change?.tier).isEqualTo(ChangeLevel.ZERO)
+    Assertions.assertThat(tier?.data?.protect?.tier).isEqualTo(ProtectLevel.A)
```

Comment:
This would test more of our code if it checked the write to the SNS topic as well. I'm not sure how easy that would be though

Reply:
Having read the rest of it, it looks like we are going to lose all of the end to end tests in this PR.

I mean the tests covering the flow from the listener to the SNS topic. I think it would be handy if we had something to give us confidence that all the bits join up properly. 

Maybe a smoke test which we run post-deployment to change a known CRN might be a better option than keeping the end to end tests here? I  do worry that with so much mocking, we can end up binding ourselves quite closely to the implementation without proving that it actually works as expected.

Reply:
Shal we bite the bullet and move away from mocked sns/sqs and run it on localstack on the build?

Reply:
I think we've a good chance of succeeding with that. It was easy on the sqs tool